src/trapfunc.cpp

```diff
@@ -53,6 +53,7 @@ static const damage_type_id damage_pure( "pure" );
 static const efftype_id effect_beartrap( "beartrap" );
 static const efftype_id effect_heavysnare( "heavysnare" );
 static const efftype_id effect_in_pit( "in_pit" );
+static const efftype_id effect_immobilization( "immobilization" );
```

Comment:
```suggestion
static const efftype_id effect_immobilization( "immobilization" );
static const efftype_id effect_in_pit( "in_pit" );
```

I am apparently terrible at the alphabet?? Fixes another clang error